mantidproject/mantid
# Update VULCAN definition

### Description of work

#### Summary of work

The VULCAN instrument at ORNL has six panels, not three.  This updates the IDF for VULCAN to reflect this fact.

#### Purpose of work

It is necessary for work on VNext that we have an accurate IDF for VULCAN inside Mantid.

#### Further detail of work

The new instrument definition file was extracted from the nexus file for `VULCAN_218075`.

### To test:

Try the following:
``` python
LoadEmptyInstrument(OutputWorkspace="wksp", InstrumentName="VULCAN")
```
then click to inspect the instrument.  It should have 6 panels.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
